lib/install/data/data.go

```diff
@@ -89,6 +89,7 @@ func NewData() *Data {
 		MappedNetworksGateways: make(map[string]net.IPNet),
 		MappedNetworksIPRanges: make(map[string][]ip.Range),
 		MappedNetworksDNS:      make(map[string][]net.IP),
+		Timeout:                3 * time.Minute,
```

Comment:
I can't see what this timeout is for, 3 minutes seems too short if it is install related.